textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcherTest.java

```diff
@@ -6,12 +6,20 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.IField;
 import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.common.Schema;
+import edu.uci.ics.textdb.common.constants.SchemaConstants;
```

Comment:
The following packages are not used (as shown by my Intellij IDE):
import edu.uci.ics.textdb.common.constants.SchemaConstants;
import edu.uci.ics.textdb.api.common.Attribute;

A general suggestion: for each Java file, use your IDE to see those unused packages and remove them.